packages/react-vanilla-components/src/components/FileUpload.tsx

```diff
@@ -26,20 +26,35 @@ import FieldWrapper from './common/FieldWrapper';
 
 const FileUpload = (props: PROPS) => {
   const fileInputField = useRef(null);
-  const { id, name, value, label, required, accept, maxFileSize, visible, enabled, appliedCssClassNames } = props;
-  let val = value && ((value instanceof Array) ? value : [value]);
+  const {
+    id,
+    name,
+    value,
+    label,
+    required,
+    accept,
+    maxFileSize,
+    visible,
+    enabled,
+    appliedCssClassNames,
+    properties
+  } = props;
+  let val = value && (value instanceof Array ? value : [value]);
   const [files, setFiles] = useState<FileObject[]>(val || []);
 
   const maxFileSizeInBytes = getFileSizeInBytes(maxFileSize);
   let newMultiple = props.type?.endsWith('[]') ? { multiple: true } : {};
 
-  const callUpdateFilesCb = useCallback((files: Array<File | FileObject>) => {
-    if (newMultiple) {
-      props.dispatchChange(files);
-    } else {
-      props.dispatchChange(files.length > 0 ? files[0] : null);
-    }
-  }, [newMultiple, props.dispatchChange]);
+  const callUpdateFilesCb = useCallback(
```

Comment:
function name  should be as per functionality

Reply:
Changed!